group-aws/README.md

```diff
@@ -9,12 +9,25 @@ There are currently no regular meetings held. The IRC channel is the
 main and official place to contact the members. For specific issues and
```

Comment:
I'm not in the Openstack Community. I'm in the Ansible AWS Working Group. 

If there are templates that come from Ansible Working Groups that are successful, then I'll take those on, otherwise I'm happy to try this approach and see what works or not.

We really don't yet have standards about what should be in this README.md.